publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponse.java

```diff
@@ -76,7 +81,16 @@ public static CdbResponse fromJson(@NonNull JSONObject json) {
       }
     }
 
-    return new CdbResponse(slots, timeToNextCall);
+    boolean consentGiven = false;
+    if (json.has(CONSENT_GIVEN)) {
+      try {
+        consentGiven = json.getBoolean(CONSENT_GIVEN);
+      } catch (JSONException ex) {
+        logger.debug("Exception while reading consentGiven" + ex.getMessage());
```

Comment:
You should not swallow an exception. Please log the whole exception:
```
logger.debug("message", ex);
```

Reply:
Btw, maybe could you apply this suggestion to the whole file ?

Reply:
Yeah was gonna say that I just copy/pasted whatever was done above :) (not that it's a good excuse).